gnosis/safe-react
# Do not send google analytics in dev

## What it solves
Resolves #2542

## How this PR fixes it
`IS_PRODUCTION` and `getCurrentEnvironment() === 'staging'` is a flag for initialising and using Google Analytics. If it is not either environment; custom dimensions, pageviews and events are instead logged to the console.

## How to test it
Load the app and view the console in the `dev` environment and see that the Google Analytics events are instead logged in the console.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/135056403-757ea36a-c1e8-40d1-b785-cad8819211a5.png)
